tf/modules/ecs_cluster/templates/ecs-setup.sh

```diff
@@ -7,3 +7,55 @@ ECS_CONTAINER_INSTANCE_TAGS=${jsonencode(ecs_cluster_tags)}
 ECS_ENABLE_TASK_IAM_ROLE=true
 EOF
 
+# Install node exporter on this machine
```

Comment:
I am not sure how ECS deployment treats it, but it's probably wise anyways, since we are running commands, to do a `set -ex` on the script so that we exit when the first command fails and prints out the full commands being executed.

Reply:
I added that line and it seems to work properly, you can check the logs in any cluster machine by checking `/var/log/cloud-init-output.log`